### DISCUSSION TITLE
[Question]: Sublabel in 0.14

### DISCUSSION THREAD

**Discussion by luisiam** [blakeblackshear/frigate#12661] [Ask A Question] [answered]
### What is your question:

I have been using ALPR to add sub labels to frigate events. Where can I see that in the new 0.14 UI?

**Comment by NickM-27** [comment]
you will see a check mark under the objects that have sub labels. You can hover to see the sub labels

<img width="152" alt="Screenshot 2024-07-29 at 9 41 02 AM" src="https://github.com/user-attachments/assets/1e2ee2c8-1dee-47c2-b081-158a74dbf27f">

**Comment by luisiam** [comment]
Probably a Safari issue but the hover popup is only shown within the snapshot and the info got cut. However, it works if I apply any filter to the events.
![Screenshot 2024-07-29 at 8 52 18 AM](https://github.com/user-attachments/assets/11778695-3314-45a6-aaf3-0f764eed7d75)

**Reply by NickM-27** [reply, marked as answer]
yes this is a safari bug, looks like an issue in safari desktop as well, fixed in https://github.com/blakeblackshear/frigate/pull/12663
